metacatalog/ext/standards_export/schemas/datacite/datacite.j2

```diff
@@ -0,0 +1,83 @@
+            <givenName>{{author.first_name}}</givenName>
+            <familyName>{{author.last_name}}</familyName>            
+            {% else %}
+            <creatorName nameType="Organizational">{{author.organisation_name}}</creatorName>
+            {% endif %}
+            {% if author.affiliation %}
+            <affiliation>{{author.affiliation}}</affiliation>
+            {% endif %}
+        </creator>
+    </creators>
+    {% endfor %}
+    <titles>
+        <title>{{title}}</title>
+    </titles>
+    <publisher>{{contact.organisationName}}</publisher>
+    <publicationYear>{{publication[0:4]}}</publicationYear>
+    {% if keywords or details%}
+    <subjects>
+        {% for detail in details %}
+        <subject>{{detail}}</subject>
+        {% endfor %}
+        {% for keyword in keywords %}
+        <subject subjectScheme="{{keyword.thesaurusName}}">{{keyword.full_path}}</subject>
+        {% endfor %}
+    </subjects>
+    {% endif %}
+    <contributors>
+        <contributor contributorType="ContactPerson">
+            <contributorName nameType="Organizational">{{contact.organisationName}}</contributorName>
+        </contributor>
+    </contributors>
+    <dates>
+        <date dateType="Created">{{publication}}</date>
+        <date dateType="Updated">{{lastUpdate}}</date>
+        {% for temporal_scale in temporal_scales %}
+        <date dateType="Collected">{{temporal_scale.temporal_extent_start}}</date>
+        <date dateType="Collected">{{temporal_scale.temporal_extent_end}}</date>
+        {% endfor %}
+    </dates>
+    <language>en</language>
+    <resourceType resourceTypeGeneral="Dataset">Dataset</resourceType>
+    <sizes />
+    <formats />
+    <version>{{version}}</version>
+    {% for license in licenses %}
+    <rightsList>
+        <rights rightsURI="{{license.link}}">{{license.short_title}}</rights>
+    </rightsList>
+    {% endfor %}
+    <descriptions>
+        <description descriptionType="Abstract"> 
+        {{abstract}}
+        </description>
+    </descriptions>
+    <geoLocations>
+        <geoLocation>
+            {% for bbox_location in bbox_locations %}
+            {% if bbox_location.min_lon == bbox_location.max_lon and bbox_location.min_lat == bbox_location.max_lat %}
+            <geoLocationPoint>
+                <pointLongitude>{{bbox_location.min_lon}}</pointLongitude>
+                <pointLatitude>{{bbox_location.min_lat}}</pointLatitude>
+            </geoLocationPoint>
+            {% else %}
+            <geoLocationBox>
+                <westBoundLongitude>{{bbox_location.min_lon}}</westBoundLongitude>
+                <eastBoundLongitude>{{bbox_location.max_lon}}</eastBoundLongitude>
+                <southBoundLatitude>{{bbox_location.min_lat}}</southBoundLatitude>
+                <northBoundLatitude>{{bbox_location.max_lat}}</northBoundLatitude>
+            </geoLocationBox>
+            {% endif %}
```

Comment:
As we allow Polygons and DataCite allows Polygons here, we can just insert a point or a polygon. No need for a bounding box